client/jetpack-connect/main-wrapper.jsx

```diff
@@ -46,7 +47,10 @@ export class JetpackConnectMainWrapper extends PureComponent {
 
 		return (
 			<Main className={ classNames( className, wrapperClassName ) }>
-				<DocumentHead title={ translate( 'Jetpack Connect' ) } />
+				<DocumentHead
+					title={ pageTitle || translate( 'Jetpack Connect' ) }
+					skipTitleFormatting={ Boolean( pageTitle ) }
```

Comment:
Just to verify: the fallback title, when `pageTitle` prop is not supplied, is supposed to be formatted? E.g., `Jetpack Connect --- WordPress.com (6)`?

Reply:
Yes, that's the existing behaviour.